hbase-client/src/main/java/org/apache/hadoop/hbase/client/MutableRegionInfo.java

```diff
@@ -206,16 +209,20 @@ public TableName getTable() {
    */
   @Override
   public boolean containsRange(byte[] rangeStartKey, byte[] rangeEndKey) {
-    if (Bytes.compareTo(rangeStartKey, rangeEndKey) > 0) {
+    CellComparator comparator = CellComparator.getComparator(tableName);
```

Comment:
Yeah, this getComparator is a dupe method? Can we have just one?

Reply:
Yep will do.